lessons/module/module-00/module-00.mdx

```diff
@@ -3,6 +3,7 @@ slug: module-0
 type: module
 next: '/tezos/storyline'
 title: Basics of blockchain
+img:../../../src/images/bot1.png
```

Comment:
@harsh242  Importing it like a component should fix the error. eg: `import image1 from "../../../src/images/bot1.png";

Reply:
Fix was to add space between `img:` and the image path.